src/rt64lib/shaders/Tracer.hlsl

```diff
@@ -331,8 +334,10 @@ void FullShadeFromGBuffers(uint hitCount, float3 rayOrigin, float3 rayDirection,
 					maxGI--;
 				}
 
+
 				// Eye light.
-				float specularIntensity = instanceProps[instanceId].materialProperties.specularIntensity;
+				// Specular is applied here. Might need to adjust the intensity of scaling for actual use cases.
+				float specularIntensity = instanceProps[instanceId].materialProperties.specularIntensity * finalSpecular;
```

Comment:
Use hitSpecular here instead so all hits have a valid specular intensity value. Otherwise, it'll get ignored for alpha hits.